src/modules/golang.rs

```diff
@@ -33,15 +34,24 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         return None;
     }
 
+    let module_version = format_go_version(&utils::exec_cmd("go", &["version"])?.stdout.as_str())?;
+
     let mut module = context.new_module("golang");
     let config: GoConfig = GoConfig::try_load(module.config);
+    let formatter = if let Ok(formatter) = StringFormatter::new(config.format) {
+        formatter.map(|variable| match variable {
+            "version" => Some(module_version.clone()),
+            _ => None,
+        })
+    } else {
+        log::warn!("Error parsing format string in `golang.format`");
+        return None;
+    };
```

Comment:
The API of `StringFormatter::map` has changed. Try merging latest `feat/format-string`.